CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.2.1 - 2017-12-06
+
+### Fixes
+
+- Fix handling of temporary read/write errors
+- Don't send initial Mixpanel events before "Anonymous Tracking" settings are loaded.
```

Comment:
Can we remove the trailing period here?

Reply:
Done

Reply:
Oh, I assumed that the CHANGELOG.md was _only_ auto-generated by `versionist`, and there was no scope for hand-editing it? If we are "allowed" to hand-edit it, then I'd suggest removing "Add a sudo-prompt upon launch on Linux-based systems." as that change actually got reverted when we removed the Linux usbboot support, and it never made it into a published version of Etcher.